pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -1295,142 +1405,65 @@ func TestSingleConstraint(t *testing.T) {
 			},
 		},
 		{
-			name: "pods spread across nodes as 2/1/0/3, maxSkew is 2, node-b and node-x fit",
```

Comment:
Why remove these tests? Seems like these tests can work well if we set proper MinDomains.

Reply:
Deletion seems to be a mistake. Revert this change.